### ISSUE TITLE
Can't mirror https repo

### ISSUE THREAD

**Issue by stuart-warren** [mikepurvis/aptly-docker#2]
At least not without having the `ca-certificates` package installed

```
# aptly mirror create -keyring=/etc/apt/trusted.gpg docker-trusty https://apt.dockerproject.org/repo ubuntu-trusty main
Downloading https://apt.dockerproject.org/repo/dists/ubuntu-trusty/Release...
ERROR: unable to fetch mirror: https://apt.dockerproject.org/repo/dists/ubuntu-trusty/Release: Get https://apt.dockerproject.org/repo/dists/ubuntu-trusty/Release: x509: failed to load system roots and no roots provided
```

I was trying to mirror the Docker repos
https://blog.docker.com/2015/07/new-apt-and-yum-repos

Thanks

**Comment by tfoote** [comment]
To resolve this error you need to install the root certificates

`apt-get update && apt-get install ca-certificates` will get you past that error.

**Comment by mikepurvis** [comment]
Resolved in 4f033498dbbad227d7ca9842d3c23e58436dfdb7.
